packages/sdk/src/clientDecryptionExtensions.ts

```diff
@@ -350,6 +350,10 @@ export class ClientDecryptionExtensions extends BaseDecryptionExtensions {
         ephemeral = false,
     }: KeyFulfilmentData & { ephemeral?: boolean }): Promise<{ error?: unknown }> {
         const userAddress = userIdToAddress(userId)
+        if (sessionIds.length === 0) {
+            this.log.debug('tried to send empty key fulfillment')
+            return {}
+        }
```

Comment:
this was a valid case, updated code here: https://github.com/towns-protocol/towns/pull/3773